src/xcode/ENA/ENA/Source/Models/User/UserMetadata.swift

```diff
@@ -0,0 +1,35 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+struct UserMetadata: Codable {
+	var federalState: String
```

Comment:
Wrong data type. I need for submission the correct `federalState`-case from the `PPAFederalState`-enum.

Reply:
Perhaps use this?
https://github.com/corona-warn-app/cwa-app-ios/blob/b3e289caad1d38a556f46efd92c17618e32a43ba/src/xcode/ENA/ENA/Source/Scenes/Datadonation/District.swift

Reply:
Changed it to FederalStateName as discussed